src/components/Person/Person.jsx

```diff
@@ -1 +1,17 @@
-// export const Person = ({ person }) => ();
+import './Person.scss';
+
+export const Person = ({ person }) => {
+  const { name, age, sex, isMarried, partnerName } = person;
+  const partnerStatus = sex === 'm' ? 'wife' : 'husband';
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">My name is {name}</h2>
+      {age !== undefined && <p className="Person__age">I am {age}</p>}
+      <p className="Person__partner">
+        {isMarried && `${partnerName} is my ${partnerStatus}`}
+        {!isMarried && 'I am not married'}
```

Comment:
Now it's enough good, but try to make code easier for reading with ternary operator, like: isMarried ? 'first condition" : "second condition"